notebooks/text-to-image-genai/text-to-image-genai.ipynb

```diff
@@ -0,0 +1,393 @@
+{
```

Comment:
<p>can you please add more explanation for this part, regarding Generator class</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2472/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
addded